src/canopy_ncf_io_mod.F90

```diff
@@ -347,7 +347,7 @@ SUBROUTINE canopy_outncf_init
         g_time%calendar = 'JULIAN'
         g_time%cartesian_axis = "T"
         g_time%long_name = 'time'
-        g_time%units = 'hours since ' // time_start
+        g_time%units = 'time steps since ' // time_start
```

Comment:
Can we have this in seconds or hours so that it is CF compliant? maybe integer steps array multiplied by the step size

Reply:
Changed.